Modules/ITS/src/ITSClusterTask.cxx

```diff
@@ -298,10 +298,6 @@ void ITSClusterTask::monitorData(o2::framework::ProcessingContext& ctx)
         if (iLayer < 3)
           for (Int_t iChip = 0; iChip < mNChipsPerHic[iLayer]; iChip++)
             hClusterSizeMonitorIB[iLayer][iStave][iChip]->Reset();
```

Comment:
you need to delete also these ones and apply the same modification as for OB (i.e. removal of single histo and include counters)